dotnet/src/webdriver/Firefox/FirefoxDriver.cs

```diff
@@ -372,7 +391,7 @@ public void UninstallAddOn(string addOnId)
         public Screenshot GetFullPageScreenshot()
         {
             Response screenshotResponse = this.Execute(GetFullPageScreenshotCommand, null);
-            string base64 = screenshotResponse.Value.ToString();
+            string base64 = screenshotResponse.Value.ToString()!;
```

Comment:
We should double-check whether `.Value` might be `null`. I mean not here, in general. Probably `Value` always not null.

Reply:
More validation is music to my ears :)

There are two real boundaries in Selenium: user input and the server. In the latter case, we basically always have an understanding of the expected response. Maybe we can have a generic `Response.GetValue<T>()` method that automatically de-serializes? Here it would look like `screenshotResponse.GetValue<string>()`. 

Or perhaps the Execute method can be generic, like so? 
```csharp
Response<string> screenshotResponse = this.Execute<string>(GetFullPageScreenshotCommand, null);
```

In this particular case, we are not actually doing null checking, since it `.Value` were null, we would get a null reference on `ToString()`. The reason we suppress the null warning is because an untyped `object.ToString()` is unfortunately annotated as nullable (almost every type I have seen replaces this with a non-nullable string).

So a "better" approach might be to cast like `string base64 = (string)screenshotResponse.Value;`, since we know the response to be a string.

Reply:
All right, this is server side. Given that old behavior is the still the same.